apollo-normalized-cache/src/commonMain/kotlin/com/apollographql/apollo/cache/normalized/MemoryCache.kt

```diff
@@ -0,0 +1,99 @@
+package com.apollographql.apollo.cache.normalized
+
+import com.apollographql.apollo.cache.ApolloCacheHeaders
+import com.apollographql.apollo.cache.CacheHeaders
+import com.apollographql.apollo.cache.normalized.internal.LruCache
+import com.apollographql.apollo.cache.normalized.internal.Platform
+import okio.internal.commonAsUtf8ToByteArray
+import kotlin.reflect.KClass
+
+class MemoryCache(
```

Comment:
Can you add a small KDoc header? I'm also saying this for myself, I don't want us to document each and every method call and parameter for a first drop of a new API that might change but adding a small description of the class sounds useful?